package.json

```diff
@@ -21,6 +21,7 @@
     ]
   },
   "scripts": {
+    "yarn": "yarn",
```

Comment:
I don't think we need this(?)

Reply:
we don't *not need this. Yarn is required in the flow and I prefer to document it.

Reply:
maybe document in a README? Agreed, seems weird to put something like that here...

Reply:
It just seems redundant. Why do we need a script called `yarn` that just runs `yarn` lol puts it in the README that running `yarn` is required for the project to work.